engine/server/server.go

```diff
@@ -179,6 +180,10 @@ type NewServerOpts struct {
 	BuildkitConfig *bkconfig.Config
 }
 
+func MyBuild(ctx context.Context, c client.Client) (_ *client.Result, err error) {
+	return nil, fmt.Errorf("FROM INSIDE MY BUILD")
```

Comment:
:eyes: testing code?

Reply:
yep... this was to get deep into buildkit to understand the differences in behavior. I will cleanup the code.